code/game/turfs/unsimulated/walls.dm

```diff
@@ -73,6 +73,8 @@
 	return null
 
 /turf/unsimulated/wall/splashscreen/New()
+	icon = 'icons/splashworks/titleBerserk.png'
+	return
```

Comment:
Comment out the rest of the code, because DreamChecker complains it is unreachable.

Reply:
yep, I just did